e2etest/pkg/frr/bgp.go

```diff
@@ -63,3 +63,26 @@ func Routes(exec executor.Executor) (map[string]Route, map[string]Route, error)
 	}
 	return v4Routes, v6Routes, nil
 }
+
+// RawDump dumps all the low leven info as a single string.
+// To be used for debugging in order to print the status of the frr instance.
+func RawDump(exec executor.Executor) (string, error) {
+	res := ""
+	out, err := exec.Exec("vtysh", "-c", "show bgp neighbor")
```

Comment:
if u have issues with bgp that will error out and prevent us from getting all other debug info maybe move that to end as best effort so we can get a chance to see running conifg and other useful info ?

Reply:
If there are issues with bgp we would still get an output, we get an error if we are really not able to execute the command.
I'll move anyhow, it shouldn't change much.

Reply:
Done